index.js

```diff
@@ -329,14 +330,16 @@ FacebookInsightStream.prototype.handleError = function ( error, retry ) {
     }
 }
 
-// predicate-based error filter 
+// predicate-based error filter
 function SkippedError ( error ) {
     return error.skip === true;
 }
 
 function errorHandler ( options, body )  {
     if ( body.error ) {
-        body.error.skip = options.ignoreMissing && body.error.code === MISSING_ERROR_CODE
+        body.error.skip = options.ignoreMissing
+        && (body.error.code === MISSING_ERROR_CODE)
+        || body.error.code === NOT_SUPPORTED_CODE
```

Comment:
Ahh ok. The would be understood more easily though with the right brackets placement
```
body.error.skip = (ignore && code === MISSING) || code === NOT_SUPPORTED
```